package.json

```diff
@@ -24,55 +24,54 @@
   },
   "homepage": "https://github.com/42BV/react-error-store#readme",
   "dependencies": {
-    "lodash.get": "^4.4.2",
-    "lodash.set": "^4.3.2"
+    "lodash": "^4.17.21"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "5.16.5",
-    "@testing-library/react": "14.0.0",
-    "@types/fetch-mock": "7.3.5",
-    "@types/jest": "29.4.0",
-    "@types/lodash.get": "4.4.7",
-    "@types/lodash.set": "4.3.7",
-    "@types/react": "18.0.28",
-    "@types/react-test-renderer": "18.0.0",
-    "@typescript-eslint/eslint-plugin": "5.52.0",
-    "@typescript-eslint/parser": "5.52.0",
-    "eslint": "8.34.0",
-    "eslint-config-prettier": "8.6.0",
+    "@testing-library/jest-dom": "6.4.1",
+    "@testing-library/react": "14.2.1",
+    "@types/fetch-mock": "7.3.8",
+    "@types/jest": "29.5.12",
+    "@types/lodash": "4.14.202",
+    "@types/react": "18.2.53",
+    "@types/react-test-renderer": "18.0.7",
+    "@typescript-eslint/eslint-plugin": "6.20.0",
+    "@typescript-eslint/parser": "6.20.0",
+    "eslint": "8.56.0",
+    "eslint-config-prettier": "9.1.0",
     "eslint-config-react-app": "7.0.1",
-    "eslint-plugin-jest": "27.2.1",
-    "eslint-plugin-prettier": "4.2.1",
-    "eslint-plugin-react": "7.32.2",
+    "eslint-plugin-jest": "27.6.3",
+    "eslint-plugin-prettier": "5.1.3",
+    "eslint-plugin-react": "7.33.2",
     "eslint-plugin-react-hooks": "4.6.0",
-    "husky": "8.0.3",
-    "jest": "29.4.3",
-    "jest-environment-jsdom": "^29.4.3",
+    "husky": "9.0.10",
+    "jest": "29.7.0",
+    "jest-environment-jsdom": "^29.7.0",
```

Comment:
Doesn't this have to be a static dependency as well? I thought the ^ was not allowed.